apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/DatabaseTestApplicationUrl.java

```diff
@@ -0,0 +1,26 @@
+package gov.cms.bfd;
+
+/** DatabaseApplicationUrl sets the application name for a jdbc url. */
+public class DatabaseTestApplicationUrl {
+  /** Database application test name constant value. */
+  public static final String DATABASE_APPLICATION_TEST_NAME = "bfdtestserver";
+  /**
+   * Sets the db url to include the application name if its not a hsqldb.
+   *
+   * @param databaseUrl database url to use
+   * @return formatted string of url with application name
+   */
+  public static String includeApplicationNameInDbUrl(String databaseUrl) {
+    if (databaseUrl.contains("jdbc:postgresql://") && !databaseUrl.contains("ApplicationName=")) {
+      if (databaseUrl.contains("?")) {
```

Comment:
I'll acccept this, but feel it is still overly complicated using String spit, etc. All you care about is:

1. Does the URL have ApplicationName
2. if not 1, then wee need to add it; then when adding, what URL delim do we need (? or &).

It doesn't matter where "ApplicationName" lives in the URL.